### DISCUSSION TITLE
Unable to test command after the latest update (1.95.1)

### DISCUSSION THREAD

**Discussion by vika-v-v** [microsoft/vscode-discussions#2381] [Extension Development QnA] [answered]
I am working on a new extension and so I ran "yo code" as always. _**The code that was automatically generated doesn't work:**_

```
// The module 'vscode' contains the VS Code extensibility API
// Import the module and reference it with the alias vscode in your code below
import * as vscode from 'vscode';

// This method is called when your extension is activated
// Your extension is activated the very first time the command is executed
export function activate(context: vscode.ExtensionContext) {

	// Use the console to output diagnostic information (console.log) and errors (console.error)
	// This line of code will only be executed once when your extension is activated
	console.log('Congratulations, your extension "angular-component-store" is now active!');

	// The command has been defined in the package.json file
	// Now provide the implementation of the command with registerCommand
	// The commandId parameter must match the command field in package.json
	const disposable = vscode.commands.registerCommand('angular-component-store.helloWorld', () => {
		// The code you place here will be executed every time your command is executed
		// Display a message box to the user
		vscode.window.showInformationMessage('Hello World from angular-component-store!');
	});

	context.subscriptions.push(disposable);
}

// This method is called when your extension is deactivated
export function deactivate() {}

``` 

```
{
  "name": "angular-component-store",
  "displayName": "angular-component-store",
  "description": "",
  "version": "0.0.1",
  "engines": {
    "vscode": "^1.95.0"
  },
  "categories": [
    "Other"
  ],
  "activationEvents": [],
  "main": "./out/extension.js",
  "contributes": {
    "commands": [
      {
        "command": "angular-component-store.helloWorld",
        "title": "Hello World"
      }
    ]
  },
  "scripts": {
    "vscode:prepublish": "npm run compile",
    "compile": "tsc -p ./",
    "watch": "tsc -watch -p ./",
    "pretest": "npm run compile && npm run lint",
    "lint": "eslint src",
    "test": "vscode-test"
  },
  "devDependencies": {
    "@types/mocha": "^10.0.9",
    "@types/node": "20.x",
    "@types/vscode": "^1.95.0",
    "@typescript-eslint/eslint-plugin": "^8.10.0",
    "@typescript-eslint/parser": "^8.7.0",
    "@vscode/test-cli": "^0.0.10",
    "@vscode/test-electron": "^2.4.1",
    "eslint": "^9.13.0",
    "typescript": "^5.6.3"
  },
  "dependencies": {
    "angular-component-store": "file:"
  }
}
``` 

The problem is that I can't call the command 'Hello World' as it is just not shown in the debugging window as I search for it.

This tutorial (https://code.visualstudio.com/api/get-started/your-first-extension) states that:
_If you aren't able to see the Hello World command in the debug window, check the package.json file and make sure that engines.vscode version is compatible with the installed version of VS Code._
So I updated the version manually to 1.95.1 and ran 'npm i' but it didn't resolve the issue.

I have also tried calling 'npm run compile' but it also didn't work.

Did someone have the same problem? Could you please help me? I need the solution as soon as possible as it blocks the development team.

**Comment by olguzzar** [comment]
Hi @vika-v-v, just want to confirm - you are able to build and run the extension, correct? A new VS Code window with "[Extension Development Host]" appears, and when you type in the command palette, the "Hello World" command does not appear?

**Reply by vika-v-v** [reply]
Hi @olguzzar, yes exactly.

**Reply by olguzzar** [reply]
I am not able to repro, the extension runs correctly for me. If you place a breakpoint in the `activate()` method and start debugging, does the breakpoint hit?

**Reply by vika-v-v** [reply]
@olguzzar unfortunately, the breakpoint doesn't fire.

**Reply by vika-v-v** [reply, marked as answer]
Update: I just found out that the problem was with the folder. I was trying to create the extension in the common repository (but it had no package.json or other files), but trying it in just C:/ worked. Still not sure what the problem was.

**Reply by olguzzar** [reply]
Thanks for the update, glad you got it working! Going to mark this discussion as answered.

**Reply by vika-v-v** [reply]
Thank you for your help!
